src/common/InlineMarkdown.tsx

```diff
@@ -13,6 +13,11 @@ export default function InlineMarkdown({
       plugins={[remarkSubSuper]}
       disallowedTypes={['paragraph']}
       unwrapDisallowed
+      renderers={{
+        paragraph: 'span',
```

Comment:
This seems redundant due `disallowedTypes` and `unwrapDisallowed`.

Reply:
@yCobanoglu how was this resolved? There are no changes and no comment explaining why it has to be so.

Reply:
ok now i removed paragraph span